### PR TITLE
Update powershell-support.md

### LINKED ISSUE #316: Links broken in powershell-support.md
The following are broken in the above-indicated file:
[Microsoft Graph PowerShell SDK](https://github.com/MicrosoftDocs/microsoft-authentication-library-dotnet/blob/main/powershell/microsoftgraph/get-started)
[Azure PowerShell SDK](https://github.com/MicrosoftDocs/microsoft-authentication-library-dotnet/blob/main/powershell/azure/new-azureps-module-az)

I glanced at the open link-checker issues and didn't see these flagged in there. Maybe they are part of the exclusions, or it's possible it's in there and I just didn't spend enough time looking.

### PR DESCRIPTION
**Pull request by pabrams**
fix two broken links

Fixes #316

### CODE REVIEW

File: msal-dotnet-articles/advanced/powershell-support.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ There is no official PowerShell module or wrapper for MSAL libraries maintained
 
 Consider using higher level APIs which are officially supported: 
 
- -  [Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started)
- -  [Azure PowerShell SDK](/powershell/azure/new-azureps-module-az)
+ -  [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell?tab=readme-ov-file)
```
Comment (gladjohn) [inline msal-dotnet-articles/advanced/powershell-support.md:12]:
```suggestion
 -  [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell)
```

this seems to be landing on the read me tab.

**Review by gladjohn — DISMISSED** [review]
thanks @pabrams

**Review by gladjohn — APPROVED** [review]

File: msal-dotnet-articles/advanced/powershell-support.md (hunk 1 above)
Comment (pabrams) [reply msal-dotnet-articles/advanced/powershell-support.md]: fixed

**Comment by gladjohn** [comment]
@pabrams, @pmaytak just pointed out that the links are working fine : https://learn.microsoft.com/en-us/entra/msal/dotnet/advanced/powershell-support

**Pull request closed without merging**